src/pages/Settings/sections/VoiceCloneSection.css

```
@@ -508,8 +508,101 @@
+  background: #667eea;
+  color: white;
+  border-color: #667eea;
+}
+
+/* Gender button active — fix font weight */
+.vc-gender-btn.active {
+  font-weight: 600;
+}
   display: flex;
   gap: 0.75rem;
 }
```

Stylelint (17.6.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)